flink-formats/flink-avro-confluent-registry/src/main/java/org/apache/flink/formats/avro/registry/confluent/RegistryAvroFormatFactory.java

```diff
@@ -175,6 +175,14 @@ public String factoryIdentifier() {
         return options;
     }
 
+    @Override
+    public Set<ConfigOption<?>> forwardOptions() {
+        Set<ConfigOption<?>> options = new HashSet<>();
+        options.addAll(requiredOptions());
```

Comment:
this looks dangerous to me if somebody adds an option above. wouldn't it be better to list them explicitly?